build.sh

```diff
@@ -155,5 +155,12 @@ configure_user(){
     echo '%wheel ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers.d/wheel-nopasswd
 }
 
+write_repo() {
+    # shellcheck source=src/cmdlib.sh
+    . "${srcdir}/src/cmdlib.sh"
+    mkdir -p /lib/coreos-assembler/src
```

Comment:
let's put these contents in a different directory if possible. Since one way to hack on coreos-assembler is to [bind mount in the src directory from the git repo to the src directory in the container](https://github.com/coreos/coreos-assembler/tree/8a445cf2cc460911d98d0cc64d4f56a20a72cd09#hacking-on-coreos-assembler-scripts) any generated files won't exist and `coreos-assembler build` will fail because `/lib/coreos-assembler/src/git.json` won't exist.

Reply:
I patched it to get it working for me locally (so I can still hack on top) and I used the  `/usr/lib/coreos-assembler-src/` directory